src/components/Reserves.js

```diff
@@ -86,6 +96,18 @@ class Reserves extends React.Component {
     this.props.history.push(`/cr/reserves/${row.courseListingId}/0/${row.id}/${row.itemId}/edit${this.props.location.search}`);
   }
 
+  getColumnMapping = () => {
```

Comment:
What is the point of breaking this out into its own function? It's only used once, right?

Reply:
You are right, it is only used once. The main reason why I did it like that is for consistency, because it's done that way in the Courses results list (and, well, in the other modules too). But I can change it from function to just an object, if you'd like, that's not a problem.

Reply:
Nah, it's fine, go ahead and merge. No need to sweat the small stuff.